x-pack/plugins/snapshot_restore/public/app/constants/index.ts

```diff
@@ -5,8 +5,12 @@
  */
 
 export const BASE_PATH = '/management/elasticsearch/snapshot_restore';
```

Comment:
Should this remain as `/snapshot_restore`?

Reply:
yeah, in order to not break bookmarks/links when we add restore functionality